docs/services/frame-system/_index.md

```diff
@@ -91,18 +93,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
+
+- You go to the **config** tab of your robot and click on the **Frame System** sub-tab.
+You have not yet added any reference frame configuration to these components, so they are all shown to be located on the default `world` origin point as follows:
+
+  ![Example robot's default frame configuration shown in the Frame System Editor. All components are stuck on top of each other](img/demo_base_unedited.png)
+
+- Then, you measure the distance on the floor from your base to your board and camera setup and find it to be 200 millimeters.
+You add this value to `"X"` in your base's reference frame `Translation` attribute and the Frame System readjusts to show the base's translation:
+
+  ![Base translated 200mm forwards shown in the Frame System Editor](img/demo_base_edited.png)
+
+- You measure the distance from your board to the camera mounted overhead and find it to be 50 millimeters.
+You add this value to `"Z"` in your camera's reference frame `Translation` attribute and the Frame System readjusts to show the camera's translation:
+
+  ![Camera translated 50 mm overhead shown in the Frame System Editor](img/demo_camera_edited_1.png)
+
+- However, the camera doesn't yet display as oriented towards the base.
+You adjust the [orientation vector](/internals/orientation-vector) 0.5 degrees in `"OX"` in your camera's reference frame `Orientation` attribute and the Frame System readjusts to show the camera's orientation:
+
+  ![Camera oriented .5 degrees OX shown in the Frame System Editor](img/demo_camera_edited_2.png)
+
+Now that the Frame System is configured so that the spatial visualization for your robot matches its spatial orientation in real life, you can utilize [Motion Service](/services/motion) methods that take in reference frame information.
+
+### Display Options
+
+Click and drag on the **Frame System** visualization to view the display from different angles, and pinch to zoom in and out:
+
+{{<gif webm_src="img/frame_system_demo.webm" mp4_src="img/frame_system_demo.mp4" alt="The frame system visualization zooming in and out around the example robot with a camera, board, and wheeled base.">}}
+
+Click the video camera icon below and to the right of the **Frame System** button to switch beween the default **Perspective Camera** and the **Orthographic Camera** view:
+
+{{< tabs name="Toggle Camera Views" >}}
+{{% tab name="Perspective Camera" %}}
+
+![Default Perspective Camera view shown in the Frame System Editor](img/demo_perspective.png)
+
+{{% /tab %}}
+{{% tab name="Orthographic Camera" %}}
+
+![Non-default Orthographic Camera view shown in the Frame System Editor](img/demo_orthographic.png)
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Bounding Geometries
+
+Add `Geometry` properties to the **Frame System** [configuration](#configuration) of a component to visualize bounding geometries by selecting a component and selecting a **Geometry** type.
+
+By default, a **Geometry** is shown surrounding the origin point of a component as follows:
```

Comment:
can you describe this? Maybe "as a cube around the component"

Reply:
it's not necessarily a box though!

Reply:
Not necessarily default as box but removed as follows